BuildLoader.py

```diff
@@ -1279,6 +1289,7 @@ def cmd_build(args):
     buildp.add_argument('-v',  '--usever',  action='store_true', help='Use board version file')
     buildp.add_argument('-fp', dest='fsppath', type=str, help='FSP binary path relative to FspBin in Silicon folder', default='')
     buildp.add_argument('-fd', '--fspdebug', action='store_true', help='Use debug FSP binary')
+    buildp.add_argument('-x64','--x64',  action='store_true', help='Build X64 image. Default is to build IA32 image.')

```

Comment:
To consider other architecture later, string argument would be extensible. 
buildp.add_argument('-a',  '--arch' , dest='arch', type=str, help='IA32, X64', default ='IA32')

Reply:
very valid point, let me change that.